docs/command-line-interface/topaz-cli/configuration.mdx

```diff
@@ -3,7 +3,7 @@ sidebar_position: 2
 ---
 # Configuration
 
-Topaz requires a configuration file to run. By default, the location of the config file is `$HOME/.config/topaz/cfg/config.yaml`. 
+Topaz requires a configuration file to run. By default, the location of the config file is `$XDG_CONFIG_HOME/topaz/cfg/config.yaml`. 
```

Comment:
XDG_CONFIG_HOME isn't defined in most people's shell. What do we use if it's not there?
Perhaps this sentence should describe the precedence order.